### ISSUE TITLE
BUG-Config attributes are not detected by Linter

### ISSUE THREAD

**Issue by AlexandreSajus** [Avaiga/taipy-core#574]
**Description**

taipy.Config attributes such as scenarios or configure_generic_data_nodes are not being detected by my Pylint linter **after loading a Config**

![Sans titre](https://github.com/Avaiga/taipy-core/assets/62465003/2fc231e4-3090-4441-aa95-ba276ef0fc24)

**How to reproduce**

Using the VSCode Pylint v2023.4.0 extension. Write the following code:
```python
from taipy import Config
Config.load("config.toml")
scenario_object = Config.scenarios["scenario"]
```
The linter should highlight that Config has no attribute scenarios

**Expected behavior**
The linter should detect every attribute of Config. This is really useful because the Linter can show me the docstrings of python functions, the type of the attributes and the code defining these attributes. These linter errors are probably linked to how Taipy defines its Config class

**Runtime environment**
 - Taipy version: 2.2.0
 - OS: Windows

**Comment by trgiangdo** [comment]
Hi @AlexandreSajus, the `Config.scenarios`, `Config.data_nodes`, ... were reported in https://github.com/Avaiga/taipy-core/issues/550. We are working on a fix for this issue.

How about the red underline at `Config.configure_generic_data_node(..`? Is it the same problem?

**Comment by AlexandreSajus** [comment]
Ah my bad, yes same issue with configure_generic_data_nodes

**Comment by trgiangdo** [comment]
Hi @AlexandreSajus, for the issue with the linter, it will be fixed in the release of Taipy 2.3. You can install the develop version and give it a try.
